Students/Semester2/lessons/students/1_basics/2_vars_and_types/2_variables_and_typestodo.ipynb

```diff
@@ -57,15 +57,15 @@
     }
    ],
    "source": [
-    "# Set a variable called name equal to your name, then set a variable of age\n",
+    "# Set a variable called name equal to your name,then set a variable of age\n",
     "# equal to your age as an integer. Print a statement with an output of\n",
     "# \"My name is [name here] and I am [age here] years old.\" using the above\n",
     "# variables in your print statement.\n",
     "\n",
     "\n",
     "# Takeaway:\n",
-    "# Concatenation (+) can only be done with strings, not strings and integers/floats. Commas, however,\n",
-    "# simply print different elements rather than attempting to concatenate strings.\n"
+    "# Concatenation (+) can only be done with strings, not strings and integers/floats. \n",
+    "# Commas, simply print different elements rather than attempting to concatenate strings.\n"
```

Comment:
This should start with "Commas (,) simply print different elements separated by a space rather than concatenating strings.